lib/holidays/date_calculator/lunisolar.rb

```diff
@@ -0,0 +1,218 @@
+module Holidays
+  module DateCalculator
+    # Copied from Korean Lunar/Solar Calendar Library, by sunsidew
+    # https://github.com/sunsidew/ruby_lunardate
```

Comment:
@sunsidew, I'm not know if it's already enough to for [BSD](https://github.com/sunsidew/ruby_lunardate/blob/master/ruby_lunardate.gemspec#L12) require?